### ISSUE TITLE
Undefined property "path"

### ISSUE THREAD

**Issue by Azayzel** [yantrashala/loopback-ssl#10]
After installing loopback and following the readme, I get 

```
Undefined property "path"
```

**Comment by slahiri** [comment]
Following is the solution. Add the following code before `return loopbackSSL.startServer(app);`

```
boot(app, __dirname, function(err) {
  if (err) throw err;
});
```

Pull request - #11

**Comment by slahiri** [comment]
Published new version 0.0.7 with README fixes.
